.github/workflows/release-subsquid-processor.yml

```diff
@@ -0,0 +1,64 @@
+name: "Release Subsquid Production Image"
+env:
+  SUBSQUID_IMAGE_RELESE_TAGS: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest,${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:{{ github.event.pull_request.head.sha }},v0.0.1
+on:
+  push:
+    tags:
+      - 'subsquid-processor-v[0-9]+\.[0-9]+\.[0-9]+'  # e.g. subsquid-processor-v0.0.1
+
+jobs:
+  test-subsquid:
+    name: Test Subquid
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Setup jest
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm ci
+        working-directory: subsquid
+      - name: Run tests
+        run: npm run test
+        working-directory: subsquid
+
+  subsquid-processor:
+    name: "Package Subsquid Processor"
+    needs:
+      - test-subsquid
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    concurrency:
+      group: ${{ github.workflow }}-package-subsquid-processor-${{ github.event.pull_request.title }}
+      cancel-in-progress: true
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          tags: ${{ env.SUBSQUID_IMAGE_RELESE_TAGS }}
+          context: subsquid
+          push: ${{ github.event_name == 'push' && github.event.pull_request.title_name == 'main' }}
```

Comment:
This seems incorrect

Reply:
I just copied pasted from `build-test-deploy.yml`  :) assumed it would work the same;
will read the doc again to what should go in there

Reply:
pushed changes to fix this, please have a look again when you have some time